secant/Resources/Localizable.strings

```diff
@@ -50,10 +50,10 @@
 
 // MARK: - Import Wallet Screen
 "importWallet.title" = "Wallet Import";
-"importWallet.description" = "You can import your backed up wallet by entering your backup recovery phrase (aka seed phrase) now.";
+"importWallet.description" = "Enter your secret backup seed phrase.";
 "importWallet.button.restoreWallet" = "Restore wallet";
 "importWallet.button.importPrivateKey" = "Import a private or viewing key";
-"importWallet.birthday.description" = "Add your wallet's birthday to sync up quicker. Leave it blank if you don't know it.";
+"importWallet.birthday.description" = "Do you know the wallet's creation date? This will allow a faster sync. If you don't know the wallet's birthday, don't worry!";
 "importWallet.birthday.placeholder" = "Enter birthday height";
```

Comment:
Can you localize the rest of the texts you changed?

Reply:
done